scripts/benchmarks_ci.py

```diff
@@ -539,5 +533,38 @@ def __main(args: list) -> int:
         # TODO: Archive artifacts.
 
 
+def __main(args: list) -> int:
+    validate_supported_runtime()
+    args = __process_arguments(args)
+    verbose = not args.quiet
+    setup_loggers(verbose=verbose)
+
+    # This validation could be cleaner
+    if args.generate_benchview_data and not args.benchview_submission_name:
+        raise RuntimeError("""In order to generate BenchView data,
+            `--benchview-submission-name` must be provided.""")
+
+    # we need to enable LongPaths support on Windows
+    if sys.platform == 'win32':
+        RunCommand(
+            'reg export HKLM\SYSTEM\CurrentControlSet\Control\FileSystem backup.reg',
+            verbose=verbose).run()
+        RunCommand(
+            'reg add HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem /v LongPathsEnabled /t REG_DWORD /d 1 /f',
+            success_exit_codes = [0, 1], # we ignore failures on purpose (Access is denied)
+            verbose=verbose).run()
+
+    try:
```

Comment:
Could you please explain these changes? They do not seem to belong here, but in the yml file.

Reply:
You are solving a lab problem, and I do not feel comfortable with changing user registry.

Reply:
> You are solving a lab problem, and I do not feel comfortable with changing user registry.

Me neither, but I am trying to solve a problem that is blocking us. How long would it take to change the settings in the lab? I am afraid that a few days at least...

Reply:
Is Arcade/Helix able to run two commands? If so, we could move the registry stuff to a `.cmd` file that runs on Windows before everything else.

Reply:
That was a very good suggestion, I have moved this logic to `HelixPreCommands` in `.yml`